collectors/python.d.plugin/adaptec_raid/README.md

```diff
@@ -6,53 +6,65 @@ sidebar_label: "Adaptec RAID"
 
 # Adaptec RAID controller monitoring with Netdata
 
-Collects logical and physical devices metrics.
+Collects logical and physical devices metrics using `arcconf` command-line utility.
+
+Executed commands:
+
+- `sudo -n arcconf GETCONFIG 1 LD`
+- `sudo -n arcconf GETCONFIG 1 PD`
 
 ## Requirements
 
-The module uses `arcconf`, which can only be executed by root.  It uses
-`sudo` and assumes that it is configured such that the `netdata` user can
-execute `arcconf` as root without password.
+The module uses `arcconf`, which can only be executed by `root`. It uses
+`sudo` and assumes that it is configured such that the `netdata` user can execute `arcconf` as root without a password.
 
-Add to `sudoers`:
+- add to the `sudoers`
 
-```
+```bash
 netdata ALL=(root)       NOPASSWD: /path/to/arcconf
 ```
 
-To grab stats it executes:
+- reset netdata systemd
+  unit [CapabilityBoundingSet](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#Capabilities) (Linux
+  distributions with systemd)
 
--   `sudo -n arcconf GETCONFIG 1 LD`
--   `sudo -n arcconf GETCONFIG 1 PD`
+Default CapabilityBoundingSet doesn't allow using `sudo` and is quite strict in general.
 
-It produces:
+> :warning: Resetting it is not an optimal solution,
+> but we couldn't find exact set of capabilities to execute arcconf with sudo.
 
-1.  **Logical Device Status**
+As the `root` user do the following:
 
-2.  **Physical Device State**
+```cmd
+mkdir /etc/systemd/system/netdata.service.d
+echo -e '[Service]\nCapabilityBoundingSet=~' | tee /etc/systemd/system/netdata.service.d/unset-capability-bounding-set.conf
+systemctl daemon-reload
+systemctl restart netdata.service
+```
 
-3.  **Physical Device S.M.A.R.T warnings**
+## Charts
 
-4.  **Physical Device Temperature**
+- Logical Device Status
+- Physical Device State
+- Physical Device S.M.A.R.T warnings
+- Physical Device Temperature
 
 ## Configuration
 
-**adaptec_raid** is disabled by default. Should be explicitly enabled in `python.d.conf`.
+`adaptec_raid` is disabled by default. Should be explicitly enabled in `python.d.conf`.
```

Comment:
@joelhans can we use `should` without any pronoun in front it to create affirmative sentences? Another doubt, can `should` to be considered a "hard" word for English speakers?

Reply:
In this case, a pronoun is necessary, but I think it would be better in this case to use ‘It can’ instead of ‘it should’. The use of ‘should’ implies a suggestion that the speaker/writer feels that it is in the listener/reader’s best interest to do what is being described, while can just indicates it as an option.